meta-mender-core/recipes-mender/mender-client/mender-client_3.5.1.bb

```diff
@@ -0,0 +1,59 @@
+require mender-client.inc
+
+################################################################################
+#-------------------------------------------------------------------------------
+# THINGS TO CONSIDER FOR EACH RELEASE:
+# - SRC_URI (particularly "branch")
+# - SRCREV
+# - DEFAULT_PREFERENCE
+#-------------------------------------------------------------------------------
+
+SRC_URI = "git://github.com/mendersoftware/mender;protocol=https;branch=3.5.x"
+
+# Tag: 3.5.1
+SRCREV = "e48c2fc825ad0c273d19f3ce00eec867091a0806"
+
+# Enable this in Betas, and in branches that cannot carry this major version as
+# default.
+# Downprioritize this recipe in version selections.
+# DEFAULT_PREFERENCE = "-1"
```

Comment:
Specifically for dunfell: You need to uncomment this line, because the 3.x major series of the client was introduced midway into the dunfell lifetime. Therefore it is not the default on this branch.